test/unit/components/VStepper/VStepperStep.spec.js

```diff
@@ -0,0 +1,18 @@
+import { test } from '@/test'
+import VStepperStep from '@/components/VStepper/VStepperStep'
+
+test('VStepperStep.js', ({ mount }) => {
+  // TODO: enable once dev merges to master
+  it.skip('should accept a custom color', async () => {
+    const wrapper = mount(VStepperStep, {
+      attachToDocument: true,
+      propsData: {
+        color: 'pink',
+        complete: true
+      }
+    })
+
+    expect(wrapper.html()).toMatchSnapshot()
+    expect('[Vue warn]: Injection "stepClick" not found').toHaveBeenWarned()
+  })
+})
```

Comment:
This is new on dev? Remove it from the PR and move it over after the next master -> dev merge (could be straight after merging this)